ballerina/Ballerina.toml

```diff
@@ -4,7 +4,7 @@ name = "ai.np"
 version = "0.4.0"
 authors = ["Ballerina"]
 keywords = ["natural programming", "ai"]
```

Comment:
With the new version of this module, we can can add compile time prompt as code, const natural expressions